server/mediaserver.go

```diff
@@ -701,7 +713,7 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 
 		// Start a watchdog to remove session after a period of inactivity
 		ticker := time.NewTicker(httpPushTimeout)
-		go func(s *LivepeerServer, mid core.ManifestID) {
+		go func(s *LivepeerServer, intmid, extmid core.ManifestID) {
```

Comment:
Only need to pass in extmid here since `removeRTMPStream()` just needs the external manifestID now.

Reply:
fixed in 63d4ab6ebadda6b3ad0f52eda2bc014d31ac5054
Need both arguments because we also need to index `rtmpConnections` inside watchdog function.

Reply:
Oh I see...Got it. Perhaps there is room for further hiding the notion of an internal manifestID in this case so a user of `rtmpConnections` only ever has to worry about the external manifestID...think that's fine to address separately.